puzzles.yaml

```diff
@@ -0,0 +1,20 @@
+app-id: uk.org.greenend.chiark.sgtatham.puzzles 
+runtime: org.gnome.Platform
+runtime-version: '43'
+sdk: org.gnome.Sdk
+command: mosaic
+modules:
+  - imagemagick.yaml
```

Comment:
there is no reason to have separate manifest for imagemagick.

Reply:
It's not use on runtime, why have everything in a single file?

Reply:
because separate manifest file a pain to maintain and should be limited to autogenerated ones (for depedencies like python / rust / npm).